tools/driver/CMakeLists.txt

```diff
@@ -89,6 +89,41 @@ swift_create_post_build_symlink(swift-frontend
   DESTINATION "swiftc${CMAKE_EXECUTABLE_SUFFIX}"
   WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
 
+swift_create_post_build_symlink(swift-frontend
+  SOURCE "swift-frontend${CMAKE_EXECUTABLE_SUFFIX}"
+  DESTINATION "sil-opt${CMAKE_EXECUTABLE_SUFFIX}"
+  WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
+swift_create_post_build_symlink(swift-frontend
+  SOURCE "swift-frontend${CMAKE_EXECUTABLE_SUFFIX}"
+  DESTINATION "sil-func-extractor${CMAKE_EXECUTABLE_SUFFIX}"
+  WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
+swift_create_post_build_symlink(swift-frontend
+  SOURCE "swift-frontend${CMAKE_EXECUTABLE_SUFFIX}"
+  DESTINATION "sil-nm${CMAKE_EXECUTABLE_SUFFIX}"
+  WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
+swift_create_post_build_symlink(swift-frontend
+  SOURCE "swift-frontend${CMAKE_EXECUTABLE_SUFFIX}"
+  DESTINATION "sil-llvm-gen${CMAKE_EXECUTABLE_SUFFIX}"
+  WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
+swift_create_post_build_symlink(swift-frontend
+  SOURCE "swift-frontend${CMAKE_EXECUTABLE_SUFFIX}"
+  DESTINATION "sil-passpipeline-dumper${CMAKE_EXECUTABLE_SUFFIX}"
+  WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
+swift_create_post_build_symlink(swift-frontend
+  SOURCE "swift-frontend${CMAKE_EXECUTABLE_SUFFIX}"
+  DESTINATION "swift-dependency-tool${CMAKE_EXECUTABLE_SUFFIX}"
+  WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
+swift_create_post_build_symlink(swift-frontend
+  SOURCE "swift-frontend${CMAKE_EXECUTABLE_SUFFIX}"
+  DESTINATION "swift-llvm-opt${CMAKE_EXECUTABLE_SUFFIX}"
+  WORKING_DIRECTORY "${SWIFT_RUNTIME_OUTPUT_INTDIR}")
+
```

Comment:
We would need to add the commands to install the symlinks, so that we don't drop those in the final toolchains.

This is a possible snippet to apply at the end of the file, together with the other installation commands.

```suggestion
swift_install_in_component(FILES "${SWIFT_RUNTIME_OUTPUT_INTDIR}/sil-opt${CMAKE_EXECUTABLE_SUFFIX}"
                           DESTINATION "bin"
                           COMPONENT tools)

swift_install_in_component(FILES "${SWIFT_RUNTIME_OUTPUT_INTDIR}/sil-func-extractor${CMAKE_EXECUTABLE_SUFFIX}"
                           DESTINATION "bin"
                           COMPONENT tools)

swift_install_in_component(FILES "${SWIFT_RUNTIME_OUTPUT_INTDIR}/sil-nm${CMAKE_EXECUTABLE_SUFFIX}"
                           DESTINATION "bin"
                           COMPONENT tools)

swift_install_in_component(FILES "${SWIFT_RUNTIME_OUTPUT_INTDIR}/sil-llvm-gen${CMAKE_EXECUTABLE_SUFFIX}"
                           DESTINATION "bin"
                           COMPONENT tools)

swift_install_in_component(FILES "${SWIFT_RUNTIME_OUTPUT_INTDIR}/sil-passpipeline-dumper${CMAKE_EXECUTABLE_SUFFIX}"
                           DESTINATION "bin"
                           COMPONENT testsuite-tools)

swift_install_in_component(FILES "${SWIFT_RUNTIME_OUTPUT_INTDIR}/swift-dependency-tool${CMAKE_EXECUTABLE_SUFFIX}"
                           DESTINATION "bin"
                           COMPONENT tools)

swift_install_in_component(FILES "${SWIFT_RUNTIME_OUTPUT_INTDIR}/swift-llvm-opt${CMAKE_EXECUTABLE_SUFFIX}"
                           DESTINATION "bin"
                           COMPONENT tools)
```

Reply:
(please double check I got the components right)

Reply:
@edymtt It's not the intention of this PR to _install_ the symlinks. We can decide later if we want this.
If someone (a compiler engineer) needs such a tool in an installed toolchain he/she can easily create the symlink.